cmd/livepeer/livepeer.go

```diff
@@ -499,6 +505,23 @@ func main() {
 	}
 }
 
+func getAuthWebhookURL(u string) (string, bool) {
+	if u == "" {
+		return "", false
+	}
+	p, err := url.ParseRequestURI(u)
+	if err != nil {
+		glog.Errorf("Error in webhook URL: %v", err)
+		return "", true
+	}
+	if p.Scheme != "http" && p.Scheme != "https" {
+		glog.Error("Webhook URL should be HTTP or HTTP")
+		return "", true
```

Comment:
`return "", fmt.Errorf("Webhook URL should be HTTP or HTTPS")` then we can avoid the logging.

Reply:
done